TARGETS

```diff
@@ -780,6 +780,7 @@ cpp_library(
         "test_util/testutil.cc",
         "tools/block_cache_analyzer/block_cache_trace_analyzer.cc",
         "tools/db_bench_tool.cc",
+        "tools/simulated_hybrid_file_system.cc",
```

Comment:
Is "env/" directory more suitable?

Reply:
I was initially under env/ and it was moved to here per @mrambacher 's suggestion. Right now it is only used in db_bench, if it is used by other components, we can rethink about it.